ui/admin/app/templates/scopes/scope/roles/role/manage-scopes/manage-custom-scopes.hbs

```diff
@@ -25,7 +25,7 @@
       }}</p>
   </page.header>
 
-  <page.body class='search-filtering'>
```

Comment:
Wow this is my bad, I totally didn't realize it would mess up the modals, the issue is [here](https://github.com/hashicorp/boundary-ui/blob/b40c53fb455f07c17bca73cd45fc7b8657317c3b/addons/core/addon/styles/addon.scss#L29) when I made the change in this [PR](https://github.com/hashicorp/boundary-ui/pull/2475/files#diff-ad9fa9adf2d80a1908951c6a64b4e36c0df232e7bed583c4ff25630b6e630a3a).

It was too permissive, does it work if we just change it to `> div`?

Reply:
Ahh how about `> div, table` 😆. I'm also open to just removing `>`, if I recall I originally had that because there could have been multiple alerts at one time for the DC changes for workday but that's not the case anymore, it's just this:

<img width="1006" alt="image" src="https://github.com/user-attachments/assets/2c7953cf-2e93-41de-b72c-9910ed874daf">

Reply:
ohh u can chain multiple together, thats cool. no need to remove it, that works. thank you!